components/admin_console/admin_definition.jsx

```diff
@@ -165,21 +168,32 @@ export const it = {
     enterpriseReady: (config, state, license, enterpriseReady) => enterpriseReady,
     licensed: (config, state, license) => license.IsLicensed === 'true',
     licensedForFeature: (feature) => (config, state, license) => license.IsLicensed && license[feature] === 'true',
+    userHasReadPermissionOnResource: (key) => (config, state, license, enterpriseReady, consoleAccess) => {
+        const keyParts = key.split('.');
```

Comment:
So this is required in the case of `user_management.something` we only want to check if the user has `read` access to the section `user_management` and not the subsection right? Might be worth commenting for future but it makes sense once you look around the code a bit so maybe not needed

Reply:
I have simplified the logic because I removed the top-level permissions for writing and reading the user_management sections.